tools/archetype/binding.test/src/main/resources/archetype-resources/src/test/java/__bindingIdCamelCase__HandlerTest.java

```diff
@@ -1,7 +1,12 @@
 #set( $dt = $package.getClass().forName("java.util.Date").newInstance() )
 #set( $year = $dt.getYear() + 1900 )
+#if( $vendorName == "Eclipse.org/SmartHome" )
```

Comment:
The vendorName is currently not set when generating a skeleton in ESH, see https://github.com/eclipse/smarthome/blob/master/extensions/binding/create_binding_skeleton.sh, so this won't actually work. Maybe you should simply check whether it is set or not?

Reply:
I beg to differ ;-) See https://github.com/eclipse/smarthome/blob/master/tools/archetype/binding/src/main/resources/META-INF/maven/archetype-metadata.xml#L8 and https://github.com/eclipse/smarthome/blob/master/tools/archetype/binding.test/src/main/resources/META-INF/maven/archetype-metadata.xml#L8

Reply:
Ok, you won 🥇